src/compiler/transformers/test/parse-events.spec.ts

```diff
@@ -105,7 +108,7 @@ describe('parse events', () => {
       references: {
         Mode: {
           location: 'local',
-          path: 'module.tsx',
+          path: IS_WINDOWS_ENV ? normalizePath('D:/module.tsx') : normalizePath('/module.tsx'),
```

Comment:
At first glance, I'm not sure I fully understand why we had to make this change. Is this due to [this change](https://github.com/ionic-team/stencil/pull/4042/files#diff-94c5db3ee58dfccae212efa88a639467f0838e1021abe6cc20cd5b2fdef7a86bR67)?

Reply:
Yes, although I just realized there's a better approach which removes the need for this